README.md

```diff
@@ -129,7 +129,23 @@ compdef kubecolor=kubectl
 
 #### fish
 
-Fish completion is unsupported because fish completion is officially not supported by kubectl.
+For fish, you can achieve completions for `kubecolor` in two steps:
```

Comment:
Can you change this sentence?
I want to emphasize that this is unofficial hack because it is not official way provided by kubernetes project.

```suggestion
Fish completion is officially unsupported by kubectl, so it is unsupported by kubecolor as well.
However, there are 2 ways we can introduce to make completion work. 
Please keep in mind these ways are a kind of "hack" and not officially supported.

1. Use "fish-kubectl-completions" with kubecolor

[evanlucas/fish-kubectl-completions](https://github.com/evanlucas/fish-kubectl-completions) is popular for fish users who wants kubectl completions in fish shell.
You can "override" kubectl with kubecolor and make it work even with kubecolor. You need to follow below 2 steps:
```